docs/content/getting-started/perform-list-users.mdx

```diff
@@ -23,7 +23,7 @@ import TabItem from '@theme/TabItem';
 # Perform a List Users call
 
 :::caution Warning
-List Users is currently an experimental feature and not yet suitable for production. There are [known limitations](#known-limitations) to be aware of. Read [the announcement](https://openfga.dev/blog/list-users-announcement) for more information.
+List Users is currently an experimental feature and not yet suitable for production. Read [the announcement](https://openfga.dev/blog/list-users-announcement) for more information.
```

Comment:
```suggestion
ListUsers is currently an experimental feature and not yet suitable for production. Read [the announcement](https://openfga.dev/blog/list-users-announcement) for more information.
```

Reply:
For better or worse, the convention in the docs is two words. Examples: "List Objects" and "List Users".

Reply:
In the caution block below we have `ListUsers` and elsewhere throughout this doc we do `List Users`. Let's just remain consistent.